wrappers/services.go

```diff
@@ -379,7 +379,7 @@ func userDaemonReload() error {
 	return cli.ServicesDaemonReload(ctx)
 }
 
-func tryFileUpdate(path string, desiredContent []byte) (old *osutil.FileState, modified bool, err error) {
+func tryFileUpdate(path string, desiredContent []byte) (old *osutil.MemoryFileState, modified bool, err error) {
```

Comment:
this is a little unfortunate that we have to pass the struct instead of the interface, but it is fine I think